evals/registry.py

```diff
@@ -146,6 +161,24 @@ def _load_registry(self, paths):
                     self._process_file(registry, path)
         return registry
 
+    def _load_registry_paths_only(self, paths, prefixes: Sequence[str] = []):
```

Comment:
ideally we could load all registries the same way, but looks like model graded specs are special in that there's only one entry per file. is it too late to change that? cc @andrew-openai

Reply:
ill fix it to follow the same way. thanks! i think we are still early and we should fix them